package.json

```diff
@@ -4,12 +4,15 @@
   "version": "1.0.0",
   "description": "Networking sockets for Electron apps",
   "license": "MIT",
-  "repository": "github:foxglove/electron-socket",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/foxglove/electron-socket.git"
+  },
   "author": {
-    "name": "Foxglove",
-    "email": "contact@foxglove.dev",
-    "url": "https://foxglove.dev/"
+    "name": "Foxglove Technologies",
+    "email": "support@foxglove.dev"
```

Comment:
prefer `contact@foxglove.dev` since support might be something specific in the future